### ISSUE TITLE
Sorry, this feature is not implemented yet!

### ISSUE THREAD

**Issue by Yakui420** [rozniak/xfce-winxp-tc#359]
Sometimes when clicking on a shortcut icon in the XP start menu, this pops up:
`Sorry, this feature is not implemented yet!`

Is it possible to edit the shortcuts manually to fix this program?
Example: When clicking on **Search**, it redirects and opens `xfce4-appfinder` for me.

**Comment by rozniak** [comment]
Yeah, those items on the Start menu are 'actions' and the code for them is in the exec library (`/shared/exec`).
